### ISSUE TITLE
Should update overwrite created if supplied

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss-labs/taco#443]
Currently, `created` is a required field in the update payload, if it differs from the existing record (as in the update example) the value is overwritten. Is it possible that someone will want to change the `created` value as part of an update? Or should update ignore whatever is provided in the update payload?

**Comment by jcoyne** [comment]
I think we should just ignore whatever is provided and keep the value from the stored record.

**Comment by aaron-collier** [comment]
Yeah, I tend to agree. My one concern would be if this isn't only a system date and could it possibly be changed. I'm assuming not, but want to make sure. 

Follow up question would be when a version changes, should the new record have a new created date or inherit the created date?
